templates/config-wb-mai6.json.jinja

```diff
@@ -1058,6 +1069,7 @@
                 "IN_P_title": "Channel P",
                 "IN_N_title": "Channel N",
                 "external_voltage_divider_description": "External voltage divider is used to measure voltage higher than 30V and to get higher input resistance. Output voltage of the divider should be in the range from 0 to 4V (better to 2V). Supported since firmware version 2.1.0."
```

Comment:
```suggestion
                "external_voltage_divider_description": "External voltage divider is used to measure voltage higher than 30V and to get higher input resistance. Output voltage of the divider should be in the range from 0 to 4V (better to 2V). Supported since firmware version 2.1.0.",
```

Reply:
OK